linux/RNGEntropy.c

```diff
@@ -0,0 +1,83 @@
+/*
+htop - RNGEntropy.c
+(C) 2022 Loic Jourdheuil
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "linux/RNGEntropy.h"
+
+#include "CRT.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/statfs.h>
+#include <sys/statvfs.h>
+
+#include "Object.h"
+#include "XUtils.h"
+
+static void getRNGEntropy(Meter* this) {
+   FILE* file = fopen(PROCDIR "/sys/kernel/random/entropy_avail", "r");
+   if (!file)
+      return;
+   int match = fscanf(file, "%le", &this->values[0]);
+   (void) match;
+   fclose(file);
+}
+
+static void getRNGPoolsize(Meter* this) {
+   FILE* file = fopen(PROCDIR "/sys/kernel/random/poolsize", "r");
+   if (!file)
+      return;
+   int match = fscanf(file, "%le", &this->total);
+   (void) match;
+   fclose(file);
+}
+
+static void RNGEntropy_updateValues(Meter* this) {
+   getRNGEntropy(this);
+   getRNGPoolsize(this);
```

Comment:
Why not have these in one function? They need to be updated in sync anyway and their code is easy enough.